Add example of good labeling (#341)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

This helps you to judge weather there are privacy concerns when sending a specific coin to some specific receiver.

![](/ReceiveLabelingRequired.png)

> You should ask yourself: "Who know this address is mine?"

Reviewer: ```suggestion
You should ask yourself: "Who knows this address is mine?"

```

An example of a good label:

> `From Max, via BitPay`

Reviewer: ```suggestion
`Max, BitPay`
```

This helps you to judge weather there are privacy concerns when sending a specific coin to some specific receiver.

![](/ReceiveLabelingRequired.png)

You should ask yourself: "Who knows this address is mine?"

An example of a good label:

> `Max, BitPay`

Reviewer 2: ```suggestion
`Alice, BTCPay`
```

We use Alice and Bob most of the time elsewhere too.
And I don't like promoting shit companies like BitPay, so we should change to something else, not sure if BTCPay works in this example though, as it is self hosted so only Alice would know of it... Do you know of any payment processor who's worthy of a shoutout?

Reviewer: > I don't like promoting shit companies like BitPay

Was thinking the same but the problem when mentioning `BTCPay` is it is not well known so readers might not understand.
How about `BTCPay (Payment processor)`?

> `Bob, Coinbase`

Reviewer 2: ```suggestion
`Bob, Bull Bitcoin`
```

Fuck conbase, they're not worthy of the shoutout, BullBitcoin on the contrary is, but they are not as well known... So I'm a bit torn, what do you say?

Reviewer: Same as above.
How about `Bull Bitcoin (Exchange)`?

This helps you to judge weather there are privacy concerns when sending a specific coin to some specific receiver.

![](/ReceiveLabelingRequired.png)

> Not only, Wasabi uses a **smart coins selection algorithm** which allows Wasabi himself to recognize entities behind a transaction and then it can automatically selects the group of coins that belong to those clusters already known.

Reviewer: Lucas said he is still working on that, so it is not there yet.

Reviewer 3: Correct, it is not ready and I don't know if I will be able to merge it into master. So, don't add it.

> Not only, Wasabi uses a **smart coins selection algorithm** which allows Wasabi himself to recognize entities behind a transaction and then it can automatically selects the group of coins that belong to those clusters already known.

Reviewer: ```suggestion
Not only, Wasabi uses a **smart coins selection algorithm** which allows Wasabi itself to recognize entities behind a transaction and then it can automatically select the group of coins that belong to those clusters already known.
```

> This allows the final user to merge more inputs from the same entities to make a transaction or partecipate in a CoinJoin round.

Reviewer: ```suggestion
This allows the final user to merge more inputs from the same entities to make a transaction or participate in a CoinJoin round.
```

This is fantastic, but it requires the final user to label in detail every address.

You should ask yourself: "Who knows this address is mine?"

> Labels should contain the comma-separated names of people/entities that may be aware about the transaction and could follow its trail like:

Reviewer: ```suggestion
Labels should contain the comma-separated names of people/entities that may be aware of the transaction and could follow its trail like:
```

`Name of the sender, name of the exchange`

So, a good label could be:

`Alice, Bitfinex`

or:

`Company Boss`

or again:

> `Alice and Bob` (that is not the same as `Alice, Bob` as they are multiple entities separated by a comma).

Reviewer: This should be `Alice & Bob`, an example of company name.
